azure-pipelines.yml

```diff
@@ -25,9 +25,6 @@ trigger:
   branches:
     include:
     - master
-  tags:
-    include:
-    - v*
```

Comment:
@carlos-villavicencio-adsk are you sure about that? Looks like all repos have this rule.

Reply:
They have it. However, we don't want to try pushing something to the app store when the tag is created since there's nothing to push in the first place and it will fail (as [it did](https://dev.azure.com/shotgun-ecosystem/1e9dee7c-030c-4741-a34e-1400feede440/_build/results?buildId=9496) with v0.1.0.dev tag).

Reply:
Toolchain is not a TK component so we don't want to do an app store release. Yes, this makes sense. Thanks for clarifying.